cpu/esp_common/esp-wifi/esp_wifi_netdev.c

```diff
@@ -799,13 +805,7 @@ static wifi_config_t wifi_config_sta = {
         .scan_method = WIFI_ALL_CHANNEL_SCAN,
         .sort_method = WIFI_CONNECT_AP_BY_SIGNAL,
         .threshold.rssi = -127,
-#if defined(MODULE_ESP_WIFI_ENTERPRISE)
-        .threshold.authmode = WIFI_AUTH_WPA2_ENTERPRISE
-#elif defined(ESP_WIFI_PASS)
-        .threshold.authmode = WIFI_AUTH_WPA_WPA2_PSK
-#else
         .threshold.authmode = WIFI_AUTH_OPEN
-#endif
```

Comment:
So no more support for encrypted networks?

Reply:
Of course, encrypted networks with authentication according to WPA2 are still supported. `.threshold.authmode` defines now the minimum requirement.